librocksdb_sys/crocksdb/c.cc

```diff
@@ -1127,6 +1131,43 @@ uint64_t crocksdb_get_snapshot_sequence_number(
   return snapshot->rep->GetSequenceNumber();
 }
 
+crocksdb_iostallinfo_t* crocksdb_create_iostalls_info() {
+  return new crocksdb_iostallinfo_t;
+}
+
+void crocksdb_destroy_iostalls_info(crocksdb_iostallinfo_t* info) {
+  delete info;
+}
+
+bool crocksdb_get_iostalls_info_cf(
```

Comment:
why not name it `crocksdb_get_map_property_cf` which is more straight forward? Same as other namings.

Reply:
It because that I just get property of `kCFStats` 
```
return db->rep->GetMapProperty(column_family->rep, DB::Properties::kCFStats, &info->rep);
```

Reply:
I agree it now, I think  rust-rocksdb should support more effective interface for TiKV